### PR TITLE
add bbcid import for timstof files

### PR DESCRIPTION
**Pull request by SteffenHeu**

### CODE REVIEW

File: mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_tdf/TDFImportTask.java [hunk 1]
```diff
@@ -391,40 +396,6 @@ private void setDescription(String desc) {
     description = desc;
   }
 
-  /**
-   * Adds all scans from the pasef segment to a raw data file. Does not add the frame spectra!
-   *
-   * @param tdfFrameTable {@link TDFFrameTable} of the tdf file
-   * @param frames        the frames to load mobility spectra for
-   */
-  private void appendScansFromTimsSegment(@Nonnull final TDFUtils tdfUtils,
```
Comment (robinschmid) [inline mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_tdf/TDFImportTask.java:400]: was this unused?

File: mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_tdf/TDFImportTask.java [hunk 2]
```diff
@@ -542,6 +513,36 @@ private void assignDiaMsMsInfo(IMSRawDataFile file, DiaFrameMsMsWindowTable diaW
     }
   }
 
+  private void assignBbCidMsMsInfo(IMSRawDataFile newMZmineFile, TDFFrameTable frameTable,
```
Comment (robinschmid) [inline mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_tdf/TDFImportTask.java:516]: maybe add javadoc to describe what BbCid is - describing the acquisition mode or data structures

File: mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_tsf/TSFImportTask.java [hunk 3]
```diff
@@ -336,4 +343,34 @@ private void addMsMsInfo(RawDataFile file) {
         "Construced " + constructed + " DDAMsMsInfos for " + file.getScans().size() + " in " + (
             end.getTime() - start.getTime()) + " ms");
   }
+
+  private void assignBbCidMsMsInfo(RawDataFile newMZmineFile, TSFFrameTable frameTable,
```
Comment (robinschmid) [inline mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_tsf/TSFImportTask.java:347]: same javadoc here?

File: mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_tdf/TDFImportTask.java (hunk 1 above)
Comment (SteffenHeu) [reply mzmine-community/src/main/java/io/github/mzmine/modules/io/import_rawdata_bruker_tdf/TDFImportTask.java:400]: yes

**Pull request merged**